airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/streams/report_streams/brands_report.py

```diff
@@ -103,7 +103,7 @@
 }
 
 METRICS_TYPE_TO_ID_MAP = {
-    "keywords": "keywordBid",
+    "keywords": "keywordId",
```

Comment:
this might just be my lack of familiarity with this connector, but given this change to use the correct `keywordId` field from a record, was this stream pretty much not functioning correctly before your fix?

Could you show an example of what a record looked like before and what it looks like now to help me understand better?